UnityPrefabs/Assets/Explosion.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Explosion : MonoBehaviour
+{
+
+    public GameObject[] particlePrefabs;
+    public int amountOfParticles = 3;
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        for (int i = 0; i < amountOfParticles; i++)
+        {
+            GameObject particlePrefab = Instantiate(particlePrefabs[Random.Range(0,particlePrefabs.Length)]);
+            particlePrefab.transform.position = transform.position;
```

Comment:
When explosion starts instantiate post explosion particles and transform their position based on prefab forces